### PR TITLE
request_source: remove default label

### PR DESCRIPTION
**Pull request by nolouch**
make the last field represent the job type,  Including the `Auto Analyze`, `TTL`, `GC` and Othres.

### CODE REVIEW

**Review by glorv — APPROVED** [review]
LGTM

File: util/request_source.go [hunk 1]
```diff
@@ -93,24 +92,26 @@ func IsRequestSourceInternal(reqSrc *RequestSource) bool {
...
+	if r.RequestSourceInternal {
+		origin = InternalRequest
+	}
+	labelList := make([]string, 0, 3)
+	labelList = append(labelList, origin)
 	if len(r.RequestSourceType) > 0 {
 		source = r.RequestSourceType
 	}
+	labelList = append(labelList, source)
 	if len(r.ExplicitRequestSourceType) > 0 {
```
Comment (glorv) [inline util/request_source.go:110]:
```suggestion
	if len(r.ExplicitRequestSourceType) > 0 && r.ExplicitRequestSourceType != r.RequestSourceType {
```
NIT: Maybe it's better to shorten duplicate request source

File: util/request_source.go [hunk 2]
```diff
@@ -30,15 +30,14 @@ const (
 // explicit source types.
 const (
 	ExplicitTypeEmpty      = ""
-	ExplicitTypeDefault    = "default"
 	ExplicitTypeLightning  = "lightning"
 	ExplicitTypeBR         = "br"
 	ExplicitTypeDumpling   = "dumpling"
 	ExplicitTypeBackground = "background"
```
Comment (glorv) [inline util/request_source.go:36]:
```suggestion
	ExplicitTypeBackground = "background"
	ExplicitTypeDDL              = "ddl"
```

File: util/request_source.go [hunk 3]
```diff
@@ -30,15 +30,14 @@ const (
 // explicit source types.
 const (
 	ExplicitTypeEmpty      = ""
-	ExplicitTypeDefault    = "default"
```
Comment (glorv) [inline util/request_source.go:33]:
```
ExplicitTypeDDL	       = "ddl"
```
Add missing type DDL

**Comment by glorv** [comment]
@Connor1996 PTAL

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**